src/client/datascience/notebook/notebookControllerManager.ts

```diff
@@ -549,4 +557,32 @@ export class NotebookControllerManager implements INotebookControllerManager, IE
             controller.dispose();
         });
     }
+
+    // Load up a python kernel to just launch in the current vscode environment
+    private loadCurrentPythonEnvironmentKernel() {
+        const config = this.configuration.getSettings();
+
+        if (config.currentPythonEnvironmentKernel) {
+            traceInfo('Load current environment kernel');
+
+            const pythonPath = config.currentPythonEnvironmentKernelPath;
+
+            // Create our KernelSpec and connection metadata that we will use to launch with
+            const currentEnvKernelSpec: IJupyterKernelSpec = {
```

Comment:
Yeah I think I wouldn't expect this if I picked 'current environment'. I'd expect it to use the active interpreter.

Reply:
This would be a good fallback if the python extension isn't installed (or active interpreter isn't set)